docs/notebooks/text_to_text.md

```diff
@@ -225,6 +226,80 @@ And finally you can use the embeded `query` to find top-k semantically related t
 query.match(index_data, limit=10, metric='cosine')
 ```
 
+<!-- #region -->
+## Before and After
+We can directly compare the results of our fine-tuned model with a its zero-shot counterpart to getter a better idea of how finetuning affects the results of a search. While the differences between the two models may be subtle for some queries, the examples below show a clear improvement in the quality of the search results:
```

Comment:
It would be nice, if we not have always the same copy-paste text. I think for quora you can probably say that the pre-trained model already performs good, but there are some queries where it returns wrong results. The finetuned model in contrast returns the correct result in most of those cases, etc.